products/argocd.md

```diff
@@ -0,0 +1,234 @@
+    latest: "1.5.8"
+    latestReleaseDate: 2020-06-17
+
+-   releaseCycle: "1.4"
+    lts: false
+    releaseDate: 2020-01-18
+    eol: 2020-04-03
+    latest: "1.4.3"
+    latestReleaseDate: 2020-04-16
+
+-   releaseCycle: "1.3"
+    lts: false
+    releaseDate: 2019-11-13
+    eol: 2020-01-18
+    latest: "1.3.6"
+    latestReleaseDate: 2019-12-11
+
+-   releaseCycle: "1.2"
+    lts: false
+    releaseDate: 2019-09-05
+    eol: 2019-11-13
+    latest: "1.2.5"
+    latestReleaseDate: 2019-10-29
+
+-   releaseCycle: "1.1"
+    lts: false
+    releaseDate: 2019-06-25
+    eol: 2019-09-05
+    latest: "1.1.2"
+    latestReleaseDate: 2019-07-31
+
+-   releaseCycle: "1.0"
+    lts: false
+    releaseDate: 2019-05-17
+    eol: 2019-06-25
+    latest: "1.0.2"
+    latestReleaseDate: 2019-06-15
+
+-   releaseCycle: "0.12"
+    lts: false
+    releaseDate: 2019-03-23
+    eol: 2019-05-17
+    latest: "0.12.3"
+    latestReleaseDate: 2019-04-30 
+
+-   releaseCycle: "0.11"
+    lts: false
+    releaseDate: 2019-01-11
+    eol: 2019-03-23
+    latest: "0.11.2"
+    latestReleaseDate: 2019-02-20  
+
+-   releaseCycle: "0.10"
+    lts: false
+    releaseDate: 2018-10-20
+    eol: 2019-01-11
+    latest: "0.10.6"
+    latestReleaseDate: 2018-11-15 
+
+-   releaseCycle: "0.9"
+    lts: false
+    releaseDate: 2018-09-25
+    eol: 2018-10-20
+    latest: "0.9.2"
+    latestReleaseDate: 2028-09-29
+
+-   releaseCycle: "0.8"
+    lts: false
+    releaseDate: 2018-09-05
+    eol: 2018-09-25
+    latest: "0.8.2"
+    latestReleaseDate: 2018-09-12 
+
+-   releaseCycle: "0.7"
+    lts: false
+    releaseDate: 2018-07-28
+    eol: 2018-09-05
+    latest: "0.7.2"
+    latestReleaseDate: 2018-08-21
+
+
+-   releaseCycle: "0.6"
+    lts: false
+    releaseDate: 2018-07-17
+    eol: 2018-07-28
+    latest: "0.6.2"
+    latestReleaseDate: 2018-07-24
+
+-   releaseCycle: "0.5"
+    lts: false
+    releaseDate: 2018-06-13
+    eol: 2018-07-17
+    latest: "0.5.4"
+    latestReleaseDate: 2018-06-28
+
+-   releaseCycle: "0.4"
+    lts: false
+    releaseDate: 2018-05-17
+    eol: 2018-06-13
+    latest: "0.4.7"
+    latestReleaseDate: 2018-06-08
+
+-   releaseCycle: "0.3"
+    lts: false
+    releaseDate: 2018-04-23
+    eol: 2018-05-17
+    latest: "0.3.2"
+    latestReleaseDate: 2018-05-04
+
+-   releaseCycle: "0.2"
+    lts: false
+    releaseDate: 2018-04-04
+    eol: 2018-04-23
+    latest: "0.2.0"
+    latestReleaseDate: 2018-04-04
+
+-   releaseCycle: "0.1"
+    lts: false
+    releaseDate: 2018-03-13
+    eol: 2018-04-04
+    latest: "0.1.0"
+    latestReleaseDate: 2018-03-13
+---
+
+> [Argo CD](https://argo-cd.readthedocs.io) is a declarative, GitOps continuous delivery tool for Kubernetes.
+
+
+- [🗓️ Release Process And Cadence](https://argo-cd.readthedocs.io/en/stable/developer-guide/release-process-and-cadence/)
+- [🎫 Argo CD roadmap](https://github.com/orgs/argoproj/projects/25/views/14)
+- [🐦 Argo CD on Twitter](https://twitter.com/argoproj)
```

Comment:
removed the twitter/roadmap links - they weren't adding much. Instead focus on the policy and cadence.

```suggestion
Only the three most recent minor versions are eligible for patch releases. Versions older than the three most recent minor versions are considered EOL and will not receive bug fixes or security updates.

## [Release Cadence](https://argo-cd.readthedocs.io/en/stable/developer-guide/release-process-and-cadence/)

A minor release occurs four times a year, once every three months. Patch releases occur on an as-needed basis.